python/mlc_chat/model/model.py

```diff
@@ -3,14 +3,14 @@
 import dataclasses
 from typing import Any, Callable, Dict, Tuple
 
-from tvm.relax.frontend import nn
-
 from mlc_chat.loader import ExternMapping, QuantizeMapping
 from mlc_chat.quantization.quantization import Quantization
+from tvm.relax.frontend import nn
```

Comment:
I feel isort is complaining this change -- my local isort has such report. Could you kindly try to move `from tvm.relax.frontend import nn` back and see if it is good?

Reply:
After moving back, it goes well. Thanks!
Because it showed that I did't pass isort check, so I used `isort /workspace/python/mlc_chat/model/model.py` to format it.